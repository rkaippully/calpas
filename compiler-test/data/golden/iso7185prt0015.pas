{

PRT test 15: Non-numeric label
             (NB: this duplicates test isoprt1760.pas)

}

program iso7185prt0015;

label one;

begin
end.