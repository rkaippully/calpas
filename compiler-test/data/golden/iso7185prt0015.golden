compiler-test/data/golden/iso7185prt0015.pas:10:7:
   |
10 | label one;
   |       ^
unexpected 'o'
expecting integer
